package.json

```diff
@@ -83,8 +83,8 @@
     "memoizee": "^0.4.15",
     "mz": "^2.7.0",
     "p-map": "^4.0.0",
-    "pinejs-client-core": "^6.13.0",
-    "pinejs-client-request": "^7.3.5",
+    "pinejs-client-core": "^6.15.10",
+    "pinejs-client-request": "^7.5.2",
```

Comment:
Since the sdk is pinned to ~6.14.0, bumping these would cause headaches in the CLI by causing it to have to separate versions of pinejs-client-core. Would it be a big deal deferring this bump for a bit (until the next sdk major)?